charts/prometheus-pushgateway/README.md

```diff
@@ -41,6 +41,10 @@ helm upgrade [RELEASE_NAME] [CHART] --install
 
 _See [helm upgrade](https://helm.sh/docs/helm/helm_upgrade/) for command documentation._
 
+### To 2.2
+
+Field `serviceMonitor.telemetryPath` (default `/metrics`) can now be configured to support changes to the telemetry path via start-up flags.
```

Comment:
imho this is not needed as this section is for major version changes

Reply:
Thank you, I will change that.